TECHNICAL_DETAILS.md

```diff
@@ -0,0 +1,13 @@
+## What are the upcoming changes?
+In Q2 2024, NEAR protocol will introduce the biggest upgrade since its mainnet launch in how the protocol operates, Stateless validation. The upgrade includes the following features:
+* Stateless validation: Validators do not have to track all shards anymore and can retrieve everything they need to validate state change (state witness) over the network. With stateless validation, NEAR can finally achieve true form of sharding and hardware requirements for a validator node can be dramatically reduced. State validation does not require local tracking of any shards as nodes can retrieve everything they need to validate state change over the network. As a result, hardware requirements for a validator node can be dramatically reduced.
```

Comment:
nit: state change -> the state transition
I would also not mention the state witness, it's an implementation detail.

nit: you wrote "hardware requirements for a validator node can be dramatically reduced." twice